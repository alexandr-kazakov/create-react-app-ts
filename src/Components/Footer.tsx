const Footer: React.FC = () => {
    return (
        <>
            <div className="bg-primary text-white">
                <div className="container mx-auto pt-10 pb-10 px-4">
                <footer>&copy; Главный одобряет</footer>
                </div>
            </div>
        </>
    )
}

export default Footer;