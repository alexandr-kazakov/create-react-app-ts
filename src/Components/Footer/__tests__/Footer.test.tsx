import { render, screen } from '@testing-library/react';
import Footer from '../Footer';

describe('Footer component', () => {
  test('renders footer with text', () => {
    render(<Footer />);
    expect(screen.getByText('© Главный одобряет')).toBeInTheDocument();
  });
});

describe('Footer component snapshot', () => {
  test('matches snapshot', () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  });
});