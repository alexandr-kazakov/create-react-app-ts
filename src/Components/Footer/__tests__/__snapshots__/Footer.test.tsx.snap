// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component snapshot matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="bg-primary text-white"
  >
    <div
      class="container mx-auto pt-10 pb-10 px-4"
    >
      <footer>
        © Главный одобряет
      </footer>
    </div>
  </div>
</DocumentFragment>
`;
